src/modules/TaskList.js

```diff
@@ -0,0 +1,66 @@
+import Task from './Task.js';
+
+export default class taskList {
+  constructor(container) {
+    this.taskArray = JSON.parse(localStorage.getItem('taskList')) || [];
+    this.container = container;
+    this.size = this.taskArray.length;
+    this.loadList();
+  }
+
+  loadList = () => {
+    const instances = [];
+    const fragment = document.createDocumentFragment();
+    this.taskArray.forEach(({ description, index, completed }) => {
+      const task = new Task(description, index, completed);
+      fragment.appendChild(this.create(task));
+      instances.push(task);
+    });
+    this.container.appendChild(fragment);
+    this.taskArray = instances;
+  }
+
+  saveLocally = () => {
+    localStorage.setItem('taskList', JSON.stringify(this.taskArray));
+  }
+
+  remove = (button) => {
+    this.taskArray = this.taskArray.filter((task) => task.index !== button.id);
+    button.parentElement.remove();
+    this.updateIndex(this.taskArray);
+    this.saveLocally();
+  }
+
```

Comment:
- [ ] Great job working on the removing functionality so far but I think you should aslo remove the list from the local storage I can see the tasks in the local storage even after deleting them so please fix this to work as expected ✨🔥💯✅ and because of that after refreshing the page I can see the tasks presist in the page even after deleting them 

- [ ] And please remeber to edit the indexes after removing the task to just keep orded from 1 to n
![image](https://user-images.githubusercontent.com/58553711/213831986-24069c16-d779-41bb-845f-4d59e37cacba.png)

![image](https://user-images.githubusercontent.com/58553711/213831895-fe3af864-9493-4239-8571-8e89a930f5b9.png)

- After refreshing the page
![image](https://user-images.githubusercontent.com/58553711/213832102-9bf54d31-b172-4be8-a4cf-b6a1783c5117.png)